kit/src/components/user_image_group/mod.rs

```diff
@@ -27,12 +27,11 @@ pub fn emit(cx: &Scope<Props>, e: Event<MouseData>) {
 #[allow(non_snake_case)]
 pub fn UserImageGroup<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
     let pressable = cx.props.onpress.is_some();
-    let count = cx.props.participants.len() as i64 - 3;
+    let count = cx.props.participants.len() as i64 - 3; // todo: why is this  - 3?
```

Comment:
We can replace this with a comment if you think it will help future people. As mentioned in private this is used because we render a maximum of 3 participants visually and the rest is added as  a "+n" later

Reply:
done